tests/callbacks/test_Manager.cpp

```diff
@@ -5,6 +5,7 @@
 
 #include "kokkos-utils/callbacks/Helpers.hpp"
 #include "kokkos-utils/impl/type_traits.hpp"
+#include "kokkos-utils/tests/fixtures/callbacks/Manager.hpp"
```

Comment:
Perhaps this include could be provided directly in "kokkos-utils/callbacks/Helpers.hpp"?

Reply:
Nope. `kokkos-utils/callbacks/Helpers.hpp` should probably be about matchers only. And it's clearly not at the right place.